src/AspNet.Security.OAuth.Nextcloud/NextcloudAuthenticationOptions.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Linq;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json.Linq;
+using static AspNet.Security.OAuth.Nextcloud.NextcloudAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.Nextcloud
+{
+    public class NextcloudAuthenticationOptions : OAuthOptions
+    {
+        public NextcloudAuthenticationOptions()
+        {
+            ClaimsIssuer = NextcloudAuthenticationDefaults.Issuer;
+            CallbackPath = new PathString(NextcloudAuthenticationDefaults.CallbackPath);
```

Comment:
This can be simplified to `CallbackPath = NextcloudAuthenticationDefaults.CallbackPath;`

Reply:
fix in next commit